examples/aws_efa/README.md

```diff
@@ -0,0 +1,157 @@
+# Using AWS Elastic Fabric Adapter (EFA) on HyperPod/EKS with SkyPilot
```

Comment:
can we add this example page to our doc page as well? https://docs.skypilot.co/en/latest/examples/frameworks/index.html